packages/react-pages/src/node/index.ts

```diff
@@ -22,17 +18,11 @@ export default function pluginFactory(
   opts: {
     pagesDir?: string
     findPages?: FindPages
-    loadPageData?: LoadPageData
```

Comment:
Is there any way we can keep `loadPageData`? I was using that :)

Reply:
Yes. I just realized we can keep the previous `addPageData`(and `loadPageData`) along with the new mutable proxy API. I will add those back.

Reply:
Done. https://github.com/vitejs/vite-plugin-react-pages/pull/19#issuecomment-808878343